docs/sources/flow/reference/components/prometheus.exporter.mssql.md

```diff
@@ -51,6 +51,21 @@ If specified, the `query_config` argument must be a YAML document as string defi
 
 See [sql_exporter](https://github.com/burningalchemist/sql_exporter#collectors) for details on how to create a configuration.
 
+### Authentication
+By default, the `USERNAME` and `PASSWORD` used within the `connection_string` should correspond with a SQL Server username and password.
```

Comment:
Should or does. Should implies it doesn't necessarily need to. If that's what is being explined here, then it's OK as-is. If it's a _must_, then we need to make sure it's clear.

```suggestion
By default, the `USERNAME` and `PASSWORD` used within the `connection_string` corresponds to an SQL Server username and password.
```